src/index.css

```diff
@@ -122,6 +130,182 @@ input:focus, .btn:active {
   color: chocolate !important;
 }
 
+.jumb {
+  margin-top: 8em;
+  background-color: rgb(210, 105, 30);
+  color:white;
+}
+
+.far:hover {
+  font-weight: 900 !important;
+}
+
+.far {
+  color: black !important;
+}
+
+.far:active {
+  font-weight: 900 !important;
+  background-color: rgb(209, 209, 209) !important;
+  color: black;
+  outline: none !important;
+  box-shadow: none !important;
+  -webkit-box-shadow: none;
+}
+
+.heading {
+  color: chocolate;
+  font-size: 2em;
+}
+
+hr {
+  margin-bottom: 40px;
+  width: 90%;
+}
+
+.profile-card {
+  width: 100%;
+  height: 17em;
+  box-shadow: 2px -2px 12px rgba(114, 107, 111, 0.21);
+  border-radius: 0;
+  margin-bottom: 20px;
+  background-color: #f3f3f3;
+  padding: 35px;
+}
+
+.profile {
+  min-height: 40vh;
+}
+
+.btn-outline-brown {
+  color: chocolate;
+  border: solid 1px chocolate;
+}
+
+.fa-map-marker {
+  padding-right: 10px;
+}
+
+.nav-bar {
+  width: 100%;
+  height: 8vh;
+  background-color: white;
+  box-shadow: 2px -2px 12px rgba(114, 107, 111, 0.21);
+}
+
+.long {
+  width: 100%;
+  margin-bottom: 15px;
+  height: 40px;
+  border: 0;
+  border-bottom: solid 1px #4c4c4c;
+  background-color: #f3f3f3;
+}
+
+input:focus {
+  outline: none;
+}
+
+.btn:focus {
+  outline: none !important;
+  box-shadow: none !important;
+}
+
+.rounded-circle {
+  border: solid 2px black;
+}
+body {
+  background-color: #f3f3f3;
+  font-family: 'Asap', sans-serif;
+}
+
+span, p, i {
+  font-family: 'Source Sans Pro', sans-serif;
+  color:#4c4c4c;
+}
+
+.heading {
+  color: chocolate;
+  font-size: 2em;
+}
+
+hr {
+  margin-bottom: 40px;
+  width: 90%;
+}
+
+.card {
```

Comment:
Please don't override the default bootstrap card class. This will make other features dependent on it display incorrectly.